<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="miNotebook_pro" failures="3" tests="4" name="Test1" time="36.073" errors="0" timestamp="2022-11-04T13:47:35 IST">
  <testcase classname="com.DWA.testcases.TC_01_LoginTest" name="loginWithValidCredentials" time="1.581"/>
  <testcase classname="com.DWA.testcases.TC_01_LoginTest" name="loginWithInvalidPassword" time="0.151">
    <failure type="java.lang.IllegalArgumentException" message="Keys to send should be a not null CharSequence">
      <![CDATA[java.lang.IllegalArgumentException: Keys to send should be a not null CharSequence
at org.openqa.selenium.remote.RemoteWebElement.sendKeys(RemoteWebElement.java:98)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:52)
at jdk.proxy2/jdk.proxy2.$Proxy29.sendKeys(Unknown Source)
at com.DWA.pageObjects.LoginPage.enterPassword(LoginPage.java:33)
at com.DWA.testcases.TC_01_LoginTest.loginWithInvalidPassword(TC_01_LoginTest.java:35)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginWithInvalidPassword -->
  <testcase classname="com.DWA.testcases.TC_01_LoginTest" name="loginWithInvalidUserId" time="0.053">
    <failure type="java.lang.IllegalArgumentException" message="Keys to send should be a not null CharSequence">
      <![CDATA[java.lang.IllegalArgumentException: Keys to send should be a not null CharSequence
at org.openqa.selenium.remote.RemoteWebElement.sendKeys(RemoteWebElement.java:98)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:52)
at jdk.proxy2/jdk.proxy2.$Proxy29.sendKeys(Unknown Source)
at com.DWA.pageObjects.LoginPage.enterUserId(LoginPage.java:28)
at com.DWA.testcases.TC_01_LoginTest.loginWithInvalidUserId(TC_01_LoginTest.java:52)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginWithInvalidUserId -->
  <testcase classname="com.DWA.testcases.TC_01_LoginTest" name="loginWithoutCredentials" time="0.37">
    <failure type="java.lang.AssertionError" message="expected [https://democonsole.dwacommerce.com/catalog/control/main] but found [https://democonsole.dwacommerce.com/catalog/control/login]">
      <![CDATA[java.lang.AssertionError: expected [https://democonsole.dwacommerce.com/catalog/control/main] but found [https://democonsole.dwacommerce.com/catalog/control/login]
at com.DWA.testcases.TC_01_LoginTest.loginWithoutCredentials(TC_01_LoginTest.java:78)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- loginWithoutCredentials -->
</testsuite> <!-- Test1 -->
